puppet-agent-alpine/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.4
 
-ENV PUPPET_VERSION="5.4.0" FACTER_VERSION="2.5.1"
+ENV PUPPET_VERSION="5.5.0" FACTER_VERSION="2.5.1"
```

Comment:
why such an old version of facter?

Reply:
Have to use Ruby facter on alpine, at least until we can get a cfacter build working

Reply:
ahh